examples/projects/dagster-slurm-example-hpc-workload/dagster_slurm_example_hpc_workload/ray/slurm-template.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```

Comment:
how is this one different to https://github.com/ascii-supply-networks/dagster-slurm/blob/main/projects/dagster-slurm-ray/dagster_slurm_ray/scripts/slurm-template.sh

does it make sense adding it here? I would prefer the other location

Reply:
But we possibly should allow the user to override and provide a custom slurm file somehow. Though, this is perhaps not a must have feature for the first MVP version

Reply:
see https://github.com/ascii-supply-networks/dagster-slurm/issues/28